v4/iacr.css

```diff
@@ -23,10 +23,11 @@
 
 
 /*** Navbar (expanded) ***/
-#iacrLogo {
+/* TODO: this doesn't appear to do anything; checking if other instances exist/actually do a thing */
+/* #iacrLogo {
   max-height: 120px;
   max-width: 120px;
-}
+} */
```

Comment:
We need to protect ourselves against changes to the logo that might result in a bigger file. I suggest leaving it here.

Reply:
Uncommented and todo remove (see commit 774533f6196b543d48f9e3546c3dc6298253475b)